src/transaction/utils/sendBatchedTransactions.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  TRANSACTION_TYPE_CALLS,
+  TRANSACTION_TYPE_CONTRACTS,
+} from '../constants';
+import type { sendBatchedTransactionsParams } from '../types';
+
+export const sendBatchedTransactions = async ({
+  capabilities,
+  sendCallsAsync,
+  transactions,
+  transactionType,
+  writeContractsAsync,
+}: sendBatchedTransactionsParams) => {
```

Comment:
Btw you did something nicer earlier in the PR, where you wrote
```
if (!transactions) {
  return;
}
```
let's continue that, as it keeps the code easier to read.